lesson12/index.js

```diff
@@ -0,0 +1,92 @@
+const ul = document.getElementById('js-list');
+const button = document.getElementById('js-button');
+
+const createLoading = () => {
+    const li = document.createElement("li");
+    const loadingImg = document.createElement('img');
+
+    li.style.listStyle = "none";
```

Comment:
use reset.css

Reply:
Moved to CSS
[ab8332a](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/12/commits/ab8332a31196cfc93765e86b503f281fce2b2e70)